app/src/main/java/sopt/org/millie/presentation/search/post/SearchPostScreen.kt

```diff
@@ -0,0 +1,61 @@
+package sopt.org.millie.presentation.search.post
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import sopt.org.millie.core.designsystem.theme.MillieTheme
+
+@Composable
+fun SearchPostScreen(
+    onClick: () -> Unit = {},
+    modifier: Modifier = Modifier,
+) {
+    LazyColumn(
+        modifier = modifier
+            .fillMaxWidth()
+            .background(MillieTheme.colors.white),
```

Comment:
저도 이거 뒷부분 배경 white인 줄 알았는데 white가 아니라 #FEFEFE여서 수정 한 번만 부탁드립니당~!!
그 컬러 세팅해놓은거보니까 background로 세팅 되어 있어서 그거 써주시면 될 것 같아요!

```suggestion
            .background(MillieTheme.colors.backgroud),
```

Reply:
앗 네넵!!